contracts/Registry.vy

```diff
@@ -115,8 +115,22 @@ def latestVault(token: address) -> address:
     return self.vaults[token][self.numVaults[token] - 1]  # dev: no vault for token
 
 
-@internal
-def _registerRelease(vault: address):
+@external
+def newRelease(vault: address):
+    """
+    @notice
+        Add a previously deployed Vault as the template contract for the latest release,
+        to be used by further "forwarder-style" delegatecall proxy contracts that can be
+        deployed from the registry throw other methods (to save gas).
+    @dev
+        Throws if caller isn't `self.governance`.
+        Throws if `vault`'s governance isn't `self.governance`.
+        Throws if the api version is the same as the previous release.
+        Emits a `NewVault` event.
+    @param vault The vault that will be used as the template contract for the next release.
+    """
+    assert msg.sender == self.governance  # dev: unauthorized
+
```

Comment:
sorry, i missed a bit of context, what was the cause for removing the governance check in the release vault?

Reply:
since we are using a proxy i can't think of a way to take advantage of a non-initialised vault as backing implementation, but it still feels off that we leave empty for any other user to initalize the implementation.

We can force to initialize to ychad.eth and stub data the implementation to brick the release just in case

Reply:
`vault` might not be initialized, and since we removed the need to endorse, there's no reason to check for an initialized parameter any more.

`apiVersion` is present in an uninitialized Vault release